src/js_realm.hpp

```diff
@@ -1734,14 +1670,15 @@ void RealmClass<T>::writeCopyTo(ContextType ctx, ObjectType this_object, Argumen
     realm::Realm::Config config;
     if (args.count == 0) {
         throw std::invalid_argument(
-            "`writeCopyTo` requires <output configuration> or <path, [encryptionKey]> parameters");
+            "`writeCopyTo` requires <output configuration> as a parameter. See documentation for details.");
```

Comment:
```suggestion
            "`writeCopyTo` requires <output configuration> as a parameter.

Reply:
I'd suggest leaving out the method name from the error message, as that will show from the callstack and is redundant as such.